cmake/config.h.cmake

```diff
@@ -33,3 +33,12 @@
 
 /* Define to 1 if you have the mm_malloc.h header */
 #cmakedefine HAVE_MM_MALLOC_H 1
+
+#cmakedefine CONFIG_PKGDATADIR "@CONFIG_PKGDATADIR@"
+
+#cmakedefine CONFIG_PREFIX "@CONFIG_PREFIX@"
```

Comment:
doesn't `@CONFIG_PREFIX@` need to be substituted in the the sed command in BUILD.bazel above?

Reply:
In fact it is unused in the whole codebase, so it is the question if it is needed at all.